### ISSUE TITLE
Cannot shift+right click anything in AE2

### ISSUE THREAD

**Issue by Ivokros** [AppliedEnergistics/Applied-Energistics-2#3710]
If I try to shift right click anything in AE2 the game treats it as a normal right click. I cannot use a memory card to save new settings or remove items with a wrench (I tried AE2, EnderIO And COFH wrenches).

#### Description
The issue is as simple as the summary. It is a simple p2p net of 4 input tunnels on 4 dense cables separately connected to the controller and 4 output tunnels to some interfaces. The network is powered via quartz fiber. Shift right clicking other items from other mods functions normally.

#### Environment
The cables are touching nothing but bare stone, they are powered, the p2p tunnels are online and powered and the network is not touching any other networks.

AE2 version is is rv5-stable-11 on Direwolf 20 1.12.2 in SMP
I do not crash and the cables function as normal but due to this I cannot sync p2p tunnels or remove cables except by using a pick.

- Minecraft Version: 1.12.2
- AE2 Version: rv5-stable-11
- Forge Version: 12.23.4.2739

**Comment by Tahakk** [comment]
Can you try it without any other mod just to be sure on items and clicking procedure? Maybe you are doing something wrong without even knowing it, if this doesn't work with AE2 only it is probably fault between table and chair, otherwise it could be other mod doing something wrong

**Comment by mindforger** [comment]
there are 2 issues which/whoose (hwo the fuck do i translate that) fix is proabbaly related to this issue (honestly not sure, just by searching for similar issues)

https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/3492
https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/3221

which both are fixed in the  rv6 alpha releases 

https://ae-mod.info/downloads/

maybe try them and see if it is fixed then (**make a copy and make sure to update other components** depending on rv5 or you will enjoy mod conflict crashes and broken worlds)

**Comment by Ivokros** [comment]
@Tahakk I used a quartz wrench from AE2 as I specified above when I said I used an AE2 wrench. The items used were all ae2 (wrench and item being shift right clicked). "(I tried AE2, EnderIO And COFH wrenches)." My main hope is the issue is just some weird server bug and it fixes itself.

@mindforger I can't update the server but I can request that the admin does so. But if it's unstable in any way I doubt they will do so. I can go break all my interfaces and replace them although I can't see the connection. I did already break and replace the cables and tunnels and the issue persisted.
